packages/core/src/anchor/ethereum/ethereum-anchor-validator.ts

```diff
@@ -171,34 +172,42 @@ export class EthereumAnchorValidator implements AnchorValidator {
       // determine network based on a chain ID
 
       const provider: providers.BaseProvider = this._getEthProvider(chainId)
-      const transaction: TransactionResponse = await this._getTransaction(provider, txHash)
-
-      if (!transaction) {
-        if (!this.ethereumRpcEndpoint) {
-          throw new Error(
-            `Failed to load transaction data for transaction ${txHash}. Try providing an ethereum rpc endpoint`
-          )
-        } else {
-          throw new Error(`Failed to load transaction data for transaction ${txHash}`)
+      const transaction: TransactionResponse = await backOff(async () => {
```

Comment:
we probably want to also set some limits on the backoff behavior.  it looks like there are arguments we can pass to limit the max number of retries and/or how long it retries for

Reply:
Backoff by default is 10 tries. Do we want to lower that?